src/AWS.Deploy.ServerMode.Client/ServerModeStartInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Security.Cryptography;
+
+namespace AWS.Deploy.ServerMode.Client
+{
+    public class ServerModeStartInfo
```

Comment:
This class does not appear to be used anymore

Reply:
removed, it is not nested in ServerModeSession itself.